Library/Homebrew/extend/pathname.rb

```diff
@@ -343,14 +343,19 @@ def ensure_writable
     chmod saved_perms if saved_perms
   end
 
+  # @private
+  def find_install_info
+    Formula["texinfo"].opt_bin/"install-info" if Formula["texinfo"].any_version_installed?
+  end
```

Comment:
```suggestion
  def install_info
    @install_info ||= begin
      if File.executable?("/usr/bin/install-info")
        "/usr/bin/install-info"
      elsif Formula["texinfo"].any_version_installed?
        Formula["texinfo"].opt_bin/"install-info" 
      end
    end
  end
```

Reply:
`brew style` complains redundant begin block, so I removed that.